playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -288,6 +288,9 @@ EDXAPP_LMS_SSL_NGINX_PORT: 48000
 EDXAPP_CMS_NGINX_PORT: 18010
 EDXAPP_CMS_SSL_NGINX_PORT: 48010
 
+# Multi site view live drop down related variable for cms
+EDXAPP_ENABLE_MULTI_SITE_VIEWLIVE_DROPDOWN: false
```

Comment:
are we planning to release this single source multi site feature? can you check with Mani and Jeff to make sure?

Reply:
i spoke with Mani last week. the tentative holdback for this is not shipping the edx-platform and edx-theme code changes. the configuration changes are so minor in the other repos (like this one)
 that it doesn't matter if we hold it back or not (and the functional outcome is the same)